content/docs/start/experiments.md

```diff
@@ -102,46 +165,56 @@ Next, run all queued experiments using `--run-all` (and in parallel with
 $ dvc exp run --run-all --jobs 2
 ```
 
+The command will run all queued experiments in a random order.
+
 ## Comparing many experiments
 
 To compare all of these experiments, we need more than `diff`. `dvc exp show`
 compares any number of experiments in one table:
 
 ```dvc
-$ dvc exp show --no-timestamp
-               --include-params train.n_est,train.min_split
-┏━━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
-┃ Experiment    ┃ avg_prec ┃ roc_auc ┃ train.n_est┃ train.min_split ┃
-┡━━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━┩
-│ workspace     │  0.56191 │ 0.93345 │ 50         │ 2               │
-│ master        │  0.55259 │ 0.91536 │ 50         │ 2               │
-│ ├── exp-bfe64 │  0.57833 │ 0.95555 │ 50         │ 8               │
-│ ├── exp-b8082 │  0.59806 │ 0.95287 │ 50         │ 64              │
-│ ├── exp-c7250 │  0.58876 │ 0.94524 │ 100        │ 2               │
-│ ├── exp-b9cd4 │  0.57953 │ 0.95732 │ 100        │ 8               │
-│ ├── exp-98a96 │  0.60405 │  0.9608 │ 100        │ 64              │
-│ └── exp-ad5b1 │  0.56191 │ 0.93345 │ 50         │ 2               │
-└───────────────┴──────────┴─────────┴────────────┴─────────────────┘
+$ dvc exp show --no-timestamp \
```

Comment:
those look a bit too complicated? can we simplify the project to keep only params that we actively show here? - just an idea

Reply:
I worked on this to find a good param-metric pair to present a more-or-less successful experiment. But at a certain level, increasing the number of units decreases the classification accuracy. Then I figured out, maybe it's rather acceptable to show an unsuccessful experiment. :)